packages/sources/svelte/src/components/VtmnTextInput.svelte

```diff
@@ -0,0 +1,77 @@
+<script>
+  import '@vtmn/css-text-input/dist/index.css';
+
+  /**
+   * ID of the input
+   * @type {string}
+   */
+  export let id;
+
+  /**
+   * Label text linked to the input
+   * @type {string}
+   */
+  export let labelText;
+
+  /**
+   * Plaholder of the input
+   * @type {string}
+   */
+  export let placeholder;
+
+  /**
+   * Whether input should be disabled or not
+   * @type {boolean}
+   */
+  export let disabled = false;
+
+  /**
+   * Helper text to help the user
+   * @type {string}
+   */
+  export let helperText;
+
+  /**
+   * Whether success should be shown or not
+   * @type {boolean}
+   */
+  export let showValid = true;
+
+  /**
+   * Whether error should be shown or not
+   * @type {boolean}
+   */
+  export let showError = true;
```

Comment:
Why do you propose props for "show"?
As you can see for example with material-ui: https://material-ui.com/components/text-fields/
They only propose one prop "error", if the consumer tells that the error is true, so they directly show the error style.
I don't really understand why there is two props :)

Reply:
The use case would be : I want to have an error management but without having the "green/red" information for the user. 
But that's fine, I removed it, let's keep it simple for now :)